README.md

```diff
@@ -17,6 +17,17 @@ A Android sample that implements Flux architecture with React and NoSQL
 }
 ```
 
+#Tests
+
+###Integration tests
+
+- The integration tests is running on top of espresso framework, and using 
+mockito to handle mocks.
+
+To setup a CI or run tests in command line use this command:
```

Comment:
To run without CI, locally...i have to call **connectCheck** gradle task? or **check**? or simply build and run?